contracts/contracts/BadgerScout.sol

```diff
@@ -190,29 +175,6 @@ contract BadgerScout is IBadgerScout, Ownable, ERC1155 {
         }
     }
 
-    /**
-     * See {IBadgerScout.setManagersBatch}
-     */
-    function setManagersBatch(
```

Comment:
I honestly cannot tell if this actually wasn't needed, but just going to roll with it.

Reply:
Confirmed duplicate logic.